docs/100.yaml

```diff
@@ -362,6 +362,9 @@
   author : Chris Hughes
 90:
   title  : Stirling’s Formula
+  links  :
+    mathlib archive: https://github.com/leanprover-community/mathlib/blob/master/archive/100-theorems-list/90_stirlings_formula.lean
```

Comment:
Since it is going to be part of mathlib, the standard formulation is rather with `decl` and the name of the theorem, see nearby entries.

Reply:
done